pkgs/applications/graphics/ImageMagick/6.x.nix

```diff
@@ -33,7 +34,7 @@ let
     else null;
 in
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: with finalAttrs; {
```

Comment:
`with finalAttrs;` shouldn't be used with `finalAttrs.finalPackage`. if `finalAttrs` is too long, `self` could be an alternative

Reply:
I was replacing `rec` with `with`, but I can do a second pass and get rid of the `with` I suppose.

Reply:
didn't review the rest of the files but this change lgtm